app/models/state_file_dependent.rb

```diff
@@ -47,18 +64,26 @@ def full_name
 
   def ask_senior_questions?
     return false if dob.nil?
-    dob <= StateFileDependent.senior_cutoff_date && months_in_home == 12 && (relationship == 'PARENT' || relationship == 'GRANDPARENT')
+    senior? && months_in_home == 12 && ['PARENT', 'GRANDPARENT'].include?(relationship)
   end
 
   def is_qualifying_parent_or_grandparent?
     ask_senior_questions? && needed_assistance_yes?
   end
 
-  def self.senior_cutoff_date
-    MultiTenantService.statefile.end_of_current_tax_year.years_ago(65)
+  def under_17?
+    age < 17
+  end
+
+  def senior?
+    age >= 65
   end
 
   def age
-    ((MultiTenantService.statefile.end_of_current_tax_year.to_time - dob.to_time) / 1.year.seconds).floor
+    MultiTenantService.statefile.current_tax_year - dob.year
+  end
```

Comment:
I had to change this because it was calculating slightly different ages on servers vs locally. Calculating age using seconds can cause some bugs because years are not the same quantity of seconds. Some years have more seconds than others. Additionally, calling `to_time` can add a timezone to a date that can adjust it to be earlier or later than the original date.